wagtail/images/tests/test_checks.py

```diff
@@ -0,0 +1,95 @@
+from unittest import TestCase
+import unittest
+from unittest.mock import mock_open, patch
+from wagtail.images.checks import has_jpeg_support
+from wagtail.images import get_image_model
+
+from .utils import Image, get_test_image_file, get_test_image_file_jpeg
+
+import pytest # type: ignore
+
+Image = get_image_model()
+@pytest.mark.django_db
+
+class TestHasJpegSupport(TestCase):
+    def setUp(self):
+        # cria uma imagem toda vez que é irá rodar um novo teste
+        self.image = Image.objects.create(
+            title="Test image",
+            file=get_test_image_file(filename="test.jpg"),
+        )
+
+    def tearDown(self):
+        #para garatir que todos os mocks sejam limpos antes da execução de um novo teste
+        patch.stopall()

```

Comment:
Issue: This shouldn't be necessary, as all mocks are used as decorators, which automatically close when the test case ends.